src/components/Box/Box.tsx

```diff
@@ -0,0 +1,60 @@
+import clsx from 'clsx';
+import styles from './Box.module.css';
+import type { FC, PropsWithChildren } from 'react';
+
+type Spacing = 'xxl' | 'xl' | 'lg' | 'md' | 'sm' | 'xs' | 'xxs';
+
+type Props = {
+  pt?: Spacing;
+  pr?: Spacing;
+  pb?: Spacing;
+  pl?: Spacing;
+  mt?: Spacing;
+  mr?: Spacing;
+  mb?: Spacing;
+  ml?: Spacing;
+  radius?: 'lg' | 'md' | 'sm' | 'xs';
+  backgroundColor?: 'primary' | 'primaryDarken' | 'accent' | 'accentDarken' | 'alert' | 'gray' | 'white';
+  border?: 'gray' | 'grayThick' | 'primary' | 'primaryThick';
```

Comment:
I'm not sure if "Thick" will get the message across.
Match the backgroundColor.